content/millennium/r4/conformance/structure-definition.md

```diff
@@ -94,6 +94,7 @@ Authorization is not required.
  [`financial-transaction-tendered-amount`]                   | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
  [`financial-transaction-type`]                              | The classification of the transaction.
  [`formatted-text`]                                          | Link to the formatted text for a note. It is used in cases when the note is not a plain text.
+ [`note-type`]                                               | Describes the type of note represented. This can be useful when there are multiple notes.
```

Comment:
Realized that we missed adding note-type to the structure definitions. As its a small change made those changes as long with the search by physical type changes but let me know if need to create a separate PR for this.

Reply:
Do we want to remove last line?

```
This can be useful when there are multiple notes.
```

Could it confuse consumer that it is only used for multiple notes scenario?

Reply:
It basically helps the consumer to determine what type of note it is, when there are multiple notes.